app/static/src/scss/includes/_accessibility.scss

```diff
@@ -26,23 +26,23 @@
   }
 
   &--accessibility {
-    color: #1d70b8;
+    color: $colour-link-visited;
     font-size: 1rem;
   }
 
   &:visited {
-    color: #1d70b8;
+    color: $colour-link-visited;
```

Comment:
$colour-link-visited is used as a main colour and visited colour on a few instances, I think these should be separate and updated with the GDS colors